packages/dds/tree/src/test/simple-tree/objectNode.spec.ts

```diff
@@ -553,5 +553,27 @@ describeHydration(
 			assert.equal(node[typeNameSymbol], Customizable.identifier);
 			assert.equal(node[typeSchemaSymbol], Customizable);
 		});
+
+		it("Build Parameter unexpected properties", () => {
+			class A extends schemaFactory.object("A", {}) {}
+			class B extends schemaFactory.object("B", { a: schemaFactory.number }) {}
+
+			const a = new A({});
+			const b = new B({ a: 1 });
+
+			// @ts-expect-error "Object literal may only specify known properties"
```

Comment:
Does the string parameter actually constrain the `@ts-expect-error` behavior? Or is it just documentation?

Reply:
Its just docs